video_xblock/static/js/studio-edit-utils.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Auxiliary functions for studio editor modal's JS.
+ */
+
+/**
+ * Prepare data to be saved to video xblock.
+ */
+function fillValues(fields) {
```

Comment:
We are sure that there aren't any conflicts now. We will provide namespaces in the next release.